MultiEditUtils.py

```diff
@@ -4,10 +4,37 @@
 
 class MultiFindAllCommand(sublime_plugin.TextCommand):
 
-  def run(self, edit, case=True, word=False, ignore_comments=False):
+  def run(self, edit, case=True, word=False, ignore_comments=False, expand=True):
 
     view = self.view
     newRegions = []
+
+    # filter selections in order to exclude duplicates since it can hang
+    # Sublime if search is performed on dozens of selections, this doesn't
+    # happen with built-in command because it works on a single selection

```

Comment:
I added this section because it was a bit hang-friendly when repeating searches over previous searches, now you can do stuff like performing multiple searches over each other for different selections and with different commands